public/js/drawing.js

```diff
@@ -6,13 +6,14 @@
     return document.getElementById(id);
   };
 
-  // var canvas = (this.__canvas = new fabric.Canvas("c", {
```

Comment:
Perhaps some of these changes toggled the drawing mode on automatically? It was working prior to this...

Reply:
Crazy, I tried just copieing what is in develop right now and the only thing that works is draw mode AND it doesn't care about line size or color. :/ I will have to work my way through this one

Reply:
I just checked out to develop and the shapes and drawing tools are working correctly... maybe something else in another file got changed?

Reply:
board controller and drawing are the only things that I can think of.

Reply:
trying...